spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/BinderCustomizer.java

```diff
@@ -34,6 +34,7 @@
 	 * In the case of a single binder, the use of this customizer is redundant.
 	 *
 	 * @param binder to be customized
+	 * @param binderName binder name to distinguish between two different instances of the same binder type
```

Comment:
A bit pedantic, perhaps, but it should say "...distinguish between multiple instances of..." (there might be more than 2).